pkg/pillar/cmd/msrv/activatecred_test.go

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) 2023 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package msrv_test
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/go-tpm/legacy/tpm2"
+	"github.com/google/go-tpm/legacy/tpm2/credactivation"
+	"github.com/lf-edge/eve/pkg/pillar/base"
+	"github.com/lf-edge/eve/pkg/pillar/cmd/msrv"
+	etpm "github.com/lf-edge/eve/pkg/pillar/evetpm"
+	"github.com/lf-edge/eve/pkg/pillar/pubsub"
+	"github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+const TpmDevicePath = "/tmp/eve-tpm/srv.sock"
```

Comment:
shouldn't this line be in `activatecred.go`?

Reply:
yes, moved things around.